.travis.yml

```diff
@@ -37,7 +41,16 @@ before_script:
 
   # Set up Go environment for build.
   - export GOPATH=$HOME/.gopath
+
   # Download godep tool and make add it to PATH environment variable.
   - go get github.com/tools/godep
   - export PATH=$PATH:$GOPATH/bin
+
+  # Download etcd and make add it to PATH environment variable.
+  - curl -fsSL https://github.com/coreos/etcd/releases/download/v2.2.1/etcd-v2.2.1-linux-amd64.tar.gz | tar xzf -
+  - ln -fns etcd-v2.2.1-linux-amd64 etcd
+  - export PATH=$PWD/etcd:$PATH
+
+  # Pull golang image
```

Comment:
Why is this needed here? Golang is already available on the VM (we're compiling Go code). 

If this is really needed, please add justification to this comment.